packages/@react-spectrum/s2/src/Calendar.tsx

```diff
@@ -10,38 +10,585 @@
+    default: 'text',
+    isDisabled: 'default'
+  }
+});
+
+
+export const Calendar = /*#__PURE__*/ (forwardRef as forwardRefType)(function Calendar<T extends DateValue>(props: CalendarProps<T>, ref: ForwardedRef<HTMLDivElement>) {
+  [props, ref] = useSpectrumContextProps(props, ref, CalendarContext);
+  let {
+    visibleMonths = 1,
+    errorMessage,
+    UNSAFE_style,
+    UNSAFE_className,
+    styles,
+    ...otherProps
+  } = props;
   return (
-    <AriaCalendar {...props}>
-      <header>
-        <Button slot="previous">◀</Button>
-        <Heading />
-        <Button slot="next">▶</Button>
-      </header>
-      <CalendarGrid>
-        {(date) => <CalendarCell date={date} />}
-      </CalendarGrid>
-      {errorMessage && <Text slot="errorMessage">{errorMessage}</Text>}
+    <AriaCalendar
+      {...otherProps}
+      ref={ref}
+      visibleDuration={{months: visibleMonths}}
+      style={UNSAFE_style}
+      className={(UNSAFE_className || '') + calendarStyles(null, styles)}>
+      {({isInvalid, isDisabled}) => {
+        return (
+          <>
+            <Header styles={headerStyles}>
+              <CalendarButton slot="previous"><ChevronLeftIcon /></CalendarButton>
+              <CalendarHeading />
+              <CalendarButton slot="next"><ChevronRightIcon /></CalendarButton>
+            </Header>
+            <div
+              className={style({
+                display: 'flex',
+                flexDirection: 'row',
+                gap: 24,
+                width: 'full'
+              })}>
+              {Array.from({length: visibleMonths}).map((_, i) => (
+                <CalendarGrid months={i} key={i} />
+              ))}
+            </div>
+            {errorMessage && (
+              <Text slot="errorMessage" className={helpTextStyles({isInvalid, isDisabled})}>
+                {/* @ts-ignore */}
+                {errorMessage}
+              </Text>
+            )}
+          </>
+        );
+      }}
     </AriaCalendar>
   );
+});
+
+export const CalendarGrid = (props: Omit<AriaCalendarGridProps, 'children'> & PropsWithChildren & {months: number}): ReactElement => {
+  // use isolation to start a new stacking context so that we can use zIndex -1 for the selection span.
+  return (
+    <AriaCalendarGrid
+      cellPadding={0}
+      className={style({
+        borderCollapse: 'collapse',
+        borderSpacing: 0,
+        isolation: 'isolate'
+      })}
+      offset={{months: props.months}}>
+      <CalendarGridHeader>
+        {(day) => (
+          <CalendarHeaderCell>
+            {day}
+          </CalendarHeaderCell>
+        )}
+      </CalendarGridHeader>
+      <CalendarGridBody>
+        {(date) => (
+          <CalendarCell date={date} firstDayOfWeek={props.firstDayOfWeek} />
+        )}
+      </CalendarGridBody>
+    </AriaCalendarGrid>
+  );
+};
+
+// Ordinarily the heading is a formatted date range, ie January 2025 - February 2025.
+// However, we want to show each month individually.
```

Comment:
This is a common request. In the future we should consider a RAC API to make this easier.